eml/components/login/RegisterForm.js

```diff
@@ -175,27 +179,6 @@ export default function RegisterForm() {
     }
   }
 
-  /**
-   * Stores the user info in async storage
-   * @param {*} userInfo: {id, firstName, lastName, email, completed courses}
-   */
-  async function saveUserInfoLocally(userInfo) {
```

Comment:
Is it safe to delete this?

Reply:
the deleted function has the same functionality as setUserInfo (line 169), which has been imported from StorageServices